.ci/docker/requirements-ci.txt

```diff
@@ -90,10 +90,10 @@ librosa>=0.6.2 ; python_version < "3.11"
 #Pinned versions:
 #test that import:
 
-mypy==1.13.0
+mypy==1.14.0
```

Comment:
Nit we should update to 1.15. It's a free massive speed boost.

Reply:
Nice! Trying it out now.

Reply:
1.15 resulted in new linter failures, so I'm going to merge this using 1.14 and do 1.15 as a separate PR so that we can lock in the improvements so far